src/LocalizedStrings.js

```diff
@@ -31,11 +31,11 @@ const placeholderRegex = /(\{[\d|\w]+\})/;
  * eg. 1: strings.formatString(strings.question, strings.bread, strings.butter)
  * eg. 2: strings.formatString(strings.question, { bread: strings.bread, butter: strings.butter }
  *
- * THIS METHOD OVERRIDE the one of the parent class by adding support for JSX code      
+ * THIS METHOD OVERRIDE the one of the parent class by adding support for JSX code
 */
 LocalizedStrings.prototype.formatString = (str, ...valuesForPlaceholders) => {
         let hasObject = false;
-        const res = str
+        const res = (str || '')
```

Comment:
I first thought this would be pulled in when updating `localized-strings`, never realised there was an override in this library. Sorry for the inconvenience.